test/kitchen.test.ts

```diff
@@ -0,0 +1,55 @@
+import test from 'ava';
+import * as cp from 'child_process';
+import * as fs from 'fs';
+import {ncp} from 'ncp';
+import * as pify from 'pify';
+import * as tmp from 'tmp';
+
+const rename = pify(fs.rename);
+const ncpp = pify(ncp);
+const keep = true;  //!!process.env.GCPM_KEEP_TEMPDIRS;
+const stagingDir = tmp.dirSync({keep, unsafeCleanup: true});
+const stagingPath = stagingDir.name;
+const pkg = require('../../package.json');
+
+const spawnp = (command: string, args: string[], options: cp.SpawnOptions = {}):
```

Comment:
Not blocking this PR, but surely a module exists for now?? if not, let's make one??

Reply:
It's a very fair question :) . I opened an issue here to discuss:
https://github.com/stephenplusplus/gcp-metadata/issues/41